src/main/java/bio/terra/janitor/service/cleanup/MetricsHelper.java

```diff
@@ -2,189 +2,131 @@
 
 import bio.terra.janitor.db.ResourceKind;
 import bio.terra.janitor.db.TrackedResourceState;
-import com.google.common.annotations.VisibleForTesting;
-import com.google.common.collect.ImmutableList;
-import io.opencensus.stats.*;
-import io.opencensus.tags.*;
+import io.opentelemetry.api.OpenTelemetry;
+import io.opentelemetry.api.common.AttributeKey;
+import io.opentelemetry.api.common.Attributes;
+import io.opentelemetry.api.metrics.DoubleHistogram;
+import io.opentelemetry.api.metrics.LongCounter;
 import java.time.Duration;
-import java.util.Arrays;
+import org.springframework.stereotype.Component;
 
 /** Helper class for recording metrics associated with cleanup. */
-class MetricsHelper {
-  private MetricsHelper() {}
-
+@Component
+public class MetricsHelper {
```

Comment:
Refactored `MetricsHelper` to use OpenTelemetry API. Really used RBS as a model.